integration-tests/api/__tests__/store/cart/cart.js

```diff
@@ -144,14 +144,17 @@ describe("/store/carts", () => {
       await dbConnection.manager.save(priceList1)
 
       const ma_sale_1 = dbConnection.manager.create(MoneyAmount, {
-        variant_id: prodSale.variants[0].id,
         currency_code: "usd",
         amount: 800,
         price_list_id: "pl_current",
       })
 
       await dbConnection.manager.save(ma_sale_1)
 
+      await dbConnection.query(
+        `INSERT INTO product_variant_money_amount(id, variant_id, money_amount_id) VALUES ('${prodSale.variants[0].id}-${ma_sale_1.id}', '${prodSale.variants[0].id}', '${ma_sale_1.id}')`
```

Comment:
comment: maybe looking at [this ](https://typeorm.io/many-to-many-relations#many-to-many-relations-with-custom-properties) would allow us to not have to do that? let me know

Reply:
question: should we do like in other places?
```
await manager.insert(ProductVariantMoneyAmount, {
  })
```